### PR TITLE
fix(README.md): adding information for local setup instructions

### PR DESCRIPTION
**Pull request by fagiani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This is the source code for our publicly viewable website, http://deis.io.
 
 If you see a typo or something outdated on our website, send us a pull request!
 
-To run the website locally for development:
+To run the website locally for development, assuming you have ruby version listed in `.ruby-version` file (see [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/)) and [bundler](http://bundler.io/) installed:
```
Comment (krancour) [inline README.md]: I would suggest rewording:

 For development, the following prerequisites must be met:

1. The ruby version indicated by `.ruby-version` must be installed. (See [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/).)
2. [bundler](http://bundler.io/) must be installed for the above mentioned ruby version.

To run the website locally:

...

**Comment by mboersma** [comment]
@fagiani I think the wording that @bacongobbler suggests is much clearer. Could you update this PR when time permits?

**Comment by mboersma** [comment]
Actually, this Deis PaaS v1 documentation repository is deprecated, and has been refactored such that the changes in this PR no longer apply. I'm going to close it, but thanks again @fagiani for helping out!

**Comment by bacongobbler** [comment]
@mboersma the documentation here is just for building deis.io itself locally, not the sphinx documentation :)

**Comment by mboersma** [comment]
@bacongobbler I know, but it's effectively mothballed now, and this PR has languished since June.

**Comment by bacongobbler** [comment]
fair enough. @fagiani please feel free to re-open another PR if you ever get back to this.
